### DISCUSSION TITLE
极空间NAS - 挂载docker.sock图文教程

### DISCUSSION THREAD

**Discussion by hslr-s** [hslr-s/sun-panel#255] [General]
> 教程来源 QQ交流3群- Bill 提供

1.因为极空间封了 portainer 的9123 端口，需要先用 lucky 把端口转发出来
![afcda568e8228a23202f32e1543e74be](https://github.com/user-attachments/assets/b9067bff-d77c-4358-b19e-219c62e46c78)
![b5d3dc705dbed67dc34f64c82c0e1797](https://github.com/user-attachments/assets/bd3ce58b-efa4-45a4-b7a4-f058f7c3f083)

2.新建远程访问，登录账号 admin，密码是需要用 md5 加密工具输入自己的极空间账号获取
![4c0b2e35588e5bb2b8dee027ae5db8d2](https://github.com/user-attachments/assets/37c213b7-7e44-4e0d-9c09-8014e9d5d726)
![2e0975bb818e9a3d00eb47096ef99e5b](https://github.com/user-attachments/assets/9256d938-4a1a-4f15-965a-46cbabd71200)
![88eccaa8b13aa99ab6b01c24dde3deaf](https://github.com/user-attachments/assets/ff3719ef-298f-4772-b392-8ae474ec344e)

3.进入 portainer 后挂载 docker.sock，重启容器
![e143bfd934a886ad630d3b50cde43f27](https://github.com/user-attachments/assets/8e21c30c-eda8-452a-889d-01c50b6aba0b)
![096fc5b441d495e591fe80f4a5bd6fc1](https://github.com/user-attachments/assets/49a793ee-3d22-41cf-a1d2-661fe1c60a9d)
![43f703d1a6c0f126c697c303bbd29b69](https://github.com/user-attachments/assets/489089fd-5b17-424a-bee3-7b9546364942)

**Comment by rollmax001** [comment]
佬，转发没用啊，还是打不开
